### ISSUE TITLE
PROD Issue: [BUG]: [Change designation of judge and court name]

### ISSUE THREAD

**Issue by muhammed-sawad** [pucardotorg/dristi#2540]
**Describe the bug**
It is showing Judge signature in order and court name showing is also not correct

**Expected behavior**
Change Judge signature to Signature, court name to 24x7 ON Court
**Screenshots**

![WhatsApp Image 2024-11-22 at 2 42 09 PM](https://github.com/user-attachments/assets/d5eb0714-4136-4706-ba9b-273c094fcb43)

Please do it fastly as we have to issue first order today itself

**Comment by Susmitabe** [comment]
![Image](https://github.com/user-attachments/assets/af04cc55-e23d-4a53-8250-cbc06cd968ad)

**Comment by Susmitabe** [comment]
Heading - In The Special Court Of Judicial Magistrate of the First Class , for the trial of cases under section 138 of NI Act , 1881 at Kollam ( "24X7 ON Court"  )

Down - Signature
            Smt. Soorya S Sukumaran
            Special Court Of Judicial Magistrate of the First Class , for the trial of cases under section 
            138 of NI Act , 1881 at Kollam ( "24X7 ON Court"  )

**Comment by bhuvanyuguru** [comment]
Hi @rajeshcherukumalli 

Tested in Uat, working fine

Refer screenshots

NOTICE

![image](https://github.com/user-attachments/assets/c25ca6ff-3e05-49d1-a05c-80e80c30e084)


![image](https://github.com/user-attachments/assets/1023b05e-e443-41ad-89bc-390d07a8db7f)


WARRANT

![image](https://github.com/user-attachments/assets/6b478454-54ff-4c79-8bfd-a61286f8ecad)



![image](https://github.com/user-attachments/assets/e869575d-3f0f-4a97-9096-cc6429f30ec5)



SUMMONS

![image](https://github.com/user-attachments/assets/62bec055-f6f3-4590-969c-58c1c96652e2)


Hearing

![image](https://github.com/user-attachments/assets/c8665795-2769-4638-9933-dd9b769e425b)

![image](https://github.com/user-attachments/assets/af8e73bb-214c-4356-8a18-af5576b07840)

**Comment by Susmitabe** [comment]
@TusharKatiyar2002   @manimaarans  is it deployed in prod ?

**Comment by Susmitabe** [comment]
@mehulseg24 , please go through the screenshots and give a signoff comment, they will deploy in production

**Comment by bhuvanyuguru** [comment]
Hi @TusharKatiyar2002, @Ramu-kandimalla , @rajeshcherukumalli 

Checked Orders for Judge, Details are shown as expected 

Refer Screenshots

![image](https://github.com/user-attachments/assets/36b6784a-3cd0-423d-b1ca-742b3bac5685)

![image](https://github.com/user-attachments/assets/7e7d74d9-069b-444d-8f99-114a25c8daf9)


![image](https://github.com/user-attachments/assets/7b4b7e11-6d87-456b-a44a-e62bc114c53e)

![image](https://github.com/user-attachments/assets/dd77df19-e015-4dc2-8718-c6d6510949b6)

**Comment by Susmitabe** [comment]
@Ajith-beehyv  & Harsh needs to confirm on summons &Warrants

**Comment by mehulseg24** [comment]
Has this been implemented for all orders? @TusharKatiyar2002 @Ramu-kandimalla

**Comment by rajeshcherukumalli** [comment]
Yes @mehulseg24 , it is updated for all orders in UAT environment

**Comment by mehulseg24** [comment]
Okay, please change the stage. Let's get the PMs to sign-off in the UAT Tahera, Anshumanth and Devaraj
